### PR TITLE
Fix connector images and pulled files

### PR DESCRIPTION
**Pull request by samgnaniah**
Fixes https://github.com/wso2/docs-apim/issues/4188

Also makes a few other fixes to broken images.

### CODE REVIEW

**Review by Mariangela — CHANGES_REQUESTED** [review]

File: en/docs/reference/connectors/exporting-artifacts.md [hunk 1]
```diff
@@ -7,17 +7,17 @@
 To bundle a Connector into a CApp, a `Connector Exporter Project` is required. 
 
 1. Navigate to **File** -> **New** -> **Other** -> **WSO2** -> **Extensions** -> **Project Types** -> **Connector Exporter Project**.<br> 
-  <img src="{{base_path}}/assets/img/integrate/connectors/connector-exporter-project-1.jpg" title="Add Connector Exporter Project" width="400" alt="Add Connector Exporter Project" />
+  <img src="../../../../assets/img/integrate/connectors/connector-exporter-project-1.jpg" title="Add Connector Exporter Project" width="400" alt="Add Connector Exporter Project" />
```
[the remark this answers is not in the document]
Comment (samgnaniah) [reply en/docs/reference/connectors/exporting-artifacts.md:10]: It seems to work correctly locally so I didn't change it.

image

 Will tab it some more and see.

File: en/docs/reference/connectors/servicenow-connector/settingup-servicenow-instance.md [hunk 2]
```diff
@@ -7,17 +7,22 @@ The ServiceNow connector allows you to access the ServiceNow REST API from an in
 1. Visit [ServiceNow](https://www.servicenow.com/) site and sign up for an account and do the verification step. 
 
 2. Now you need to create an instance. To do that visit [ServiceNow Developer account](https://developer.servicenow.com/dev.do) with the credentials you obtained when following the previous step. Click on the **Request Instance** button. 
+  <br>
   <img src="{{base_path}}/assets/img/integrate/connectors/servicenow-2.png" title="Login to Developer Account" width="800" alt="Login to Developer Account"/>
```
[the remark this answers is not in the document]
Comment (samgnaniah) [reply en/docs/reference/connectors/servicenow-connector/settingup-servicenow-instance.md]: Fixed

File: en/docs/reference/connectors/exporting-artifacts.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (samgnaniah) [reply en/docs/reference/connectors/exporting-artifacts.md]: Fixed.

File: en/docs/reference/connectors/exporting-artifacts.md [hunk 3]
```diff
@@ -7,17 +7,17 @@
...
 
 1. Navigate to **File** -> **New** -> **Other** -> **WSO2** -> **Extensions** -> **Project Types** -> **Connector Exporter Project**.<br> 
-  <img src="{{base_path}}/assets/img/integrate/connectors/connector-exporter-project-1.jpg" title="Add Connector Exporter Project" width="400" alt="Add Connector Exporter Project" />
+   <img src="../../../../assets/img/integrate/connectors/connector-exporter-project-1.jpg" title="Add Connector Exporter Project" width="400" alt="Add Connector Exporter Project" />
 
 2. Enter a name for the Connector Exporter Project. 
 
 3. In the next screen select, **Specify the parent from workspace** and select the specific Integration Project you created from the dropdown. 
-  <img src="{{base_path}}/assets/img/integrate/connectors/connector-exporter-project-naming.png" title="Naming Connector Exporter Project" width="400" alt="Naming Connector Exporter Project" />
+   <img src="../../../../assets/img/integrate/connectors/connector-exporter-project-naming.png" title="Naming Connector Exporter Project" width="400" alt="Naming Connector Exporter Project" />
```
Comment (Mariangela) [inline en/docs/reference/connectors/exporting-artifacts.md:15]: @samgnaniah , why have you removed base_path?

**Review by Mariangela — CHANGES_REQUESTED** [review]

File: en/docs/reference/connectors/exporting-artifacts.md (hunk 3 above)
Comment (samgnaniah) [reply en/docs/reference/connectors/exporting-artifacts.md:15]: base_path doesn't work for include pages hence the change here. These were all broken due to base_path used here - maybe we have changed it accidentally at some point.

**Review by Mariangela — APPROVED** [review]

**Pull request merged**